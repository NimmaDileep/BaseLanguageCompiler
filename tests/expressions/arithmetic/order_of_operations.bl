print(5 * 3 - 21 + 7 - 3 * 2);
print(5 * (3 - 21) + (7 - 3) * 2);
print((5.5 * 3) - 2.6 + 7 - (6.8 * 2));
