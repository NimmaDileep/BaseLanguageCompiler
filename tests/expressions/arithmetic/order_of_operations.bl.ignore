print(5 * 3 - 21 + 7 - 3 * 2);
print(5 * (3 - 21) + (7 - 3) * 2);
