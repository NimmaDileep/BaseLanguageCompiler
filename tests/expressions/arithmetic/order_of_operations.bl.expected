-5
-82
7.299999